Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Tokenization.hs

```diff
@@ -0,0 +1,45 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Domain.Action.UI.Tokenization where
+
+import qualified API.Types.UI.Tokenization
+import Data.OpenApi (ToSchema)
+import qualified Domain.Types.Merchant
+import qualified Domain.Types.Merchant.MerchantOperatingCity
+import qualified Domain.Types.Merchant.MerchantServiceConfig as DomainMSC
+import qualified Domain.Types.Person
+import qualified Environment
+import EulerHS.Prelude hiding (id)
+import qualified Kernel.External.Tokenize.HyperVerge.Types as HyperVergeTypes
```

Comment:
When Token is generic then why importing Hyperverge types? It could be any tokenization service, right?